dev/bots/check_code_samples.dart

```diff
@@ -11,6 +11,7 @@ import 'package:args/args.dart';
 import 'package:file/file.dart';
 import 'package:file/local.dart';
 import 'package:path/path.dart' as path;
+import 'package:string_scanner/string_scanner.dart';
```

Comment:
In the pubspec we should move this to be an explicit dependency, not just a dependency of a dependency.

Reply:
It's actually unused (added by the IDE), so I removed it.